.scannerwork/report-task.txt

```diff
@@ -0,0 +1,6 @@
+projectKey=DEFRA_biodiversity-net-gain-service
+serverUrl=http://127.0.0.1:9000
+serverVersion=9.6.1.59531
+dashboardUrl=http://127.0.0.1:9000/dashboard?id=DEFRA_biodiversity-net-gain-service
+ceTaskId=AYPNI6HjXAHgg2375MjL
+ceTaskUrl=http://127.0.0.1:9000/api/ce/task?id=AYPNI6HjXAHgg2375MjL
```

Comment:
Does this file and/or .scannerwork/.sonarlock contain any sensitive values that require the file to be removed from version control history and the regeneration of sensitive values accordingly?

Reply:
No.